src/Controls/src/Core/Handlers/Items/StructuredItemsViewHandler.Windows.cs

```diff
@@ -17,6 +19,32 @@ public partial class StructuredItemsViewHandler<TItemsView> : ItemsViewHandler<T
 
 		protected override IItemsLayout Layout { get => ItemsView?.ItemsLayout; }
 
+		protected override void ConnectHandler(ListViewBase platformView)
+		{
+			base.ConnectHandler(platformView);
+
+			if (Layout != null)
```

Comment:
Move to is not over is.
```suggestion
			if (Layout is not null)
```